src/vsr/replica.zig

```diff
@@ -183,12 +183,13 @@ pub fn ReplicaType(
         /// * never decreases while the replica is alive
         commit_min: u64,
 
-        /// The op number of the latest committed operation (according to the cluster):
+        /// The op number of the latest committed operation (according to the cluster).
         /// This is the commit number in terms of the VRR paper.
         ///
         /// Invariants:
         /// * `replica.commit_max ≥ replica.commit_min`.
-        /// * never decreases
+        /// * `replica.commit_max = replica.commit_min` for status=normal primary.
+        /// * never decreases.
         commit_max: u64,
```

Comment:
When reviewing this code, remember that:

- `commit_max` _may exceed the head op_, but
- the head op must _never_ exceed `op_checkpoint_trigger`.